packages/flame/test/components/component_test.dart

```diff
@@ -45,6 +45,36 @@ void main() {
   final prepareGame = FlameTester(() => _PrepareGame());
 
   group('Component', () {
+    testWithFlameGame('children in the constructor', (game) async {
+      game.add(
+        Component(
+          children: [ComponentA(), ComponentB()],
+        ),
+      );
+      await game.ready();
+
+      expect(game.children.length, 1);
+      expect(game.children.first.children.length, 2);
+      expect(game.children.first.children.elementAt(0), isA<ComponentA>());
+      expect(game.children.first.children.elementAt(1), isA<ComponentB>());
+    });
+
+    testWithFlameGame('children in constructor and onLoad', (game) async {
+      final component = TwoChildrenComponent(
+        children: [ComponentA(), ComponentB()],
+      );
+      game.add(component);
+      await game.ready();
```

Comment:
ensureAdd?

Reply:
`game.ready()` does the same, without the need to use a special extension.

Reply:
You do write one line leas though